ansible-satellite6/ansible_runner/project/partition_disk.yml

```diff
@@ -0,0 +1,19 @@
+---
+- hosts: all
+  remote_user: root
+  tasks:
+    - name: partition-disk
+      shell: df -P /home | awk 'END{print $NF}'
+      register: disk_free
+
+    - debug: msg="{{ disk_free.stdout }}"
+    
+    - shell: |
+        umount /home
```

Comment:
Why not use https://docs.ansible.com/ansible/latest/modules/parted_module.html

Reply:
Great feedback. Chris is fairly new to ansible. So, like myself, he probably isn't aware of all the modules available to make some of these things easier.
@chbrown13 This would be a great module to look into!

Reply:
Ok cool, will take a look. Thanks @bkearney

Reply:
Might want to look at this:
https://docs.ansible.com/ansible/latest/modules/lvg_module.html?highlight=logical%20volumes

Since we actually aren't partitioning the disk, but removing and resizing logical volumes. Might want to come up with a better term for this too something.

Reply:
Also I think we want to make sure we have a directory structure that mimics what galaxy would use in most cases.  Because that will make it easier is we want to contribute upstream to galaxy as well.  "ansible-galaxy init --offline -p roles rolename" is very useful as it will create a empty galaxy directory structure.

Reply:
@flatfender that change will be made, if needed, when we discuss our plans for this repo on Friday. The current structure is to be compatible with ansible-runner.